bitmovin/getting-started.md

```diff
@@ -56,3 +56,12 @@ After creating the instance of the Bitmovin player, simply pass that player refe
   });
 </script>
 ```
+
+## Confirming it works
+
+After you've finished integration, you can quickly see if it's working by opening your [browser developer tools](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_are_browser_developer_tools), selecting the network tab, and then clicking 'play' on the video you've set to be monitored. You should begin to see beacons being sent to the mux servers (litix.io) like in the example image.
```

Comment:
Yeah, I can see the confusion, but the mozilla page is the best overview of all tools I can find, not just specific to FF.